src/app/case.service.ts

```diff
@@ -8,7 +8,7 @@ export class CaseService {
   constructor(private httpClient: HttpClient) { }
 
 
-  fetch(): Observable<Object> {
-        return this.httpClient.get('http://localhost:3000/api/cases/bob');
+  fetch(caseId): Observable<Object> {
+        return this.httpClient.get(`http://localhost:3000/api/cases/${caseId}`);
```

Comment:
Can this be relative? So it works deployed.

Reply:
Yes definitely but would like to do it as a dedicated piece of work that thinks about environments and universal data fetching